hpx/runtime/serialization/serialize_sequence.hpp

```diff
@@ -1,173 +0,0 @@
-//  Copyright (c) 2007-2015 Hartmut Kaiser
```

Comment:
I am not sure if we should remove it entirely. Users might still want to be able to serialize fusion sequences.